src/EnergyPlus/MixedAir.cc

```diff
@@ -452,35 +452,35 @@ void SimOAComponent(EnergyPlusData &state,
     {
         auto const SELECT_CASE_var(CompTypeNum);
 
```

Comment:
If `CompTypeNum` is already of the right type, then you can just use that instead of `SELECT_CASE_var` and `static_cast<int>`.

Reply:
Thanks for the feedback. I'll fix this.